common/common.go

```diff
@@ -50,3 +50,9 @@ func IsGHError(err error) bool {
 	_, ok := err.(*github.ErrorResponse)
```

Comment:
Comment on this method:
```suggestion
// IsGHError checks  if any errror from github is returned
```